DBSCE/Overview.md

```diff
@@ -0,0 +1,281 @@
+
+### DBSC(E) use cases
+
+This section expands on the [generic design](#high-level-design) to address different enterprise use cases:
+
+#### IDP is RP and Calls Public Local Key Helper
+
+This is the same use case elaborated in the high level design [above](#high-level-design). However, we have separated the IdP/RP components of the server for the enterprise use case in the below diagram.
+
+![IDPSameAsRP-CallsPublicLocalKeyHelper](./IDPSameAsRP-CallsPublicLocalKeyHelper.svg)
+
+Highlights:
+
+- Auth tokens are not mentioned in the DBSC(E) high level design to simplify the over all scenario. However, most signin/access operations will often make use of identity token. In such case, the IdP can generate the auth tokens and embed the thumbprint of the public key in the token.
+- The token can be delivered to the RP directly through an API instead of a header based response.
+- The tokens also contain the thumpbrint of the public key, and the RP must validate the thumbprint from the token against the thumbprint from the JWT.
+
+#### IDP Calls Public Local Key Helper
+
+In many use cases, it is also a valid to have separate servers as [RP](#relying-party-rp) and [IdP](#identity-provider-idp). We address the use case where these are separate entities and probably from different vendors below.
+
+For easy mapping with the existing DBSC proposal, please note:
+
+- Steps 1-16 specify the key generation process for a public local key helper.
+- Steps 17-29 are [DBSC](https://github.com/wicg/dbsc), added for completeness.
+
+![IDPCallsPublicLocalKeyHelper](./IDPCallsPublicLocalKeyHelper.svg)
+
+#### IDP Calls Private Local Key Helper
+
+A special case is for enterprises that already have `well-known` Local Key Helpers, which are expected to be trusted and enabled by default in a browser. Here, since the browser can trust a given IDP (based on the URL and a policy mapping) and can trust the IDP to invoke the appropriate Local Key Helper (refer [Local key helper on Windows](./KeyGeneration.md#local-key-helper-on-windows)), there are a few optimizations that can be made to the protocol, in skipping the `nonce` and reducing the number of round trips between the IDP and the Local Key Helper.
+
+![IDPCallsPrivateLocalKeyHelper](./IDPCallsPrivateLocalKeyHelper.svg)
+
+A _private local key helper_ is a special use case. In this case the IDP owns the local key helper [implementation](#local-key-helper), and can use a private protocol to communicate with the local key helper.
+
+Highlights:
+
+
+
+- Since proof of device is made with SSO headers, the browser can skip the `nonce` and directly call the Local Key Helper.
```

Comment:
SSO headers is the implementation detail. We should highlight that proof of device is what matters, e.g.

- Since IDP uses a proprietary mechanism to perform device authentication during the sign-in process, binding statement doesn't require additional nonce, because IDP already ensures that request is coming from a correct device. A common mechanism to do such a device authentication is to use short-lived SSO headers with nonce, shown on the diagram.

Reply:
This is not yet reworked, I will add a "in draft" for this section